Maddori.Apple/Maddori.Apple/Screen/Setup/InvitedCodeViewController.swift

```diff
@@ -0,0 +1,87 @@
+//
+//  InvitedCodeViewController.swift
+//  Maddori.Apple
+//
+//  Created by 이성호 on 2022/10/23.
+//
+
+import UIKit
+
+import SnapKit
+
+final class InvitedCodeViewController: BaseViewController {
+    
+    // MARK: - property
+    
+    private let titleLabel: UILabel = {
+        let label = UILabel()
+        label.setTitleFont(text: TextLiteral.invitedCodeViewControllerTitleLabel)
+        return label
+    }()
+    private let invitedCodeLabel: UILabel = {
+        let label = UILabel()
+        label.text = "1BCDFF"
+        label.font = UIFont.systemFont(ofSize: 32, weight: .bold)
```

Comment:
p1;

Font적용이 안된 것 같아요 !
피그마엔 Apple SD Gothic Neo로 설정되어있더라구요 !
<img width="540" alt="image" src="https://user-images.githubusercontent.com/78677571/197554129-cd61fb3e-6a75-4a86-be77-753ca9aead57.png">

```suggestion
        label.font = UIFont.font(.bold, ofSize: 32)
```

Reply:
[04832f5](https://github.com/DeveloperAcademy-POSTECH/MacC-Team-Maddori.Apple/pull/39/commits/04832f5edeafcd224708cc511bc6e7d3cc20e4a7)

수정완료옴